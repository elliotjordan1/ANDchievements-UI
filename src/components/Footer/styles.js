import styled from 'styled-components';
import theme from '../../global/theme';

export const FooterWrapper = styled.div`
  display: flex;
  -webkit-align-items: center;
  justify-content: center;
  align-self: flex-end;
  background-color: ${theme.colours.brand.red};
  position: relative;
  bottom: 0;
  width: 100%;
  height: 30px;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
  @media ${theme.media.tablet} {
    height: 50px;
  };
  @media ${theme.media.desktop} {
    height: 65px;
  };
`;

export const FooterText = styled.a`
  display: flex;
  text-decoration: none;
  align-items: center;
  width: 100%;
  font-family: ${theme.fontFamily.primary};
  font-style: normal;
  font-weight: normal;
  font-size: 24px;
  line-height: 140%;
  text-align: center;
  letter-spacing: 0.1em;
  color: #fff;
  height: 20px;
  margin-left: 12px;
  cursor: pointer;
  font-size: 10px;
  @media ${theme.media.tablet} {
    height: 30px;
    font-size: 14px;
  };
  @media ${theme.media.desktop} {
    height: 40px;
    font-size: 18px;
  };
`;

export const FooterDiv = styled.div`
  display: flex;
`;