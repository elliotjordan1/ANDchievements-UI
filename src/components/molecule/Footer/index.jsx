import React from 'react';
import {
  FooterWrapper,
  FooterText,
  FooterDiv
} from '../../atom';

const Footer = () => {
  return (
    <FooterWrapper>
      <FooterDiv>
            <FooterText href="https://and.digital/about/" text ="About" />
      </FooterDiv>
      <FooterDiv>
            <FooterText text = "/" />
      </FooterDiv>
      <FooterDiv>
            <FooterText href="https://and.digital/contact/" text = "Contact Us" />
      </FooterDiv>
      <FooterDiv>
            <FooterText text = "/" />
      </FooterDiv>
      <FooterDiv>
            <FooterText href="https://and.digital/join-us/" text = "Join Us" />
      </FooterDiv>
      <FooterDiv>
            <FooterText text = "/" />
      </FooterDiv>
      <FooterDiv>
            <FooterText href="https://and.digital/" text = "AND Digital" />
      </FooterDiv>
    </FooterWrapper>
  );
};

export default Footer;
