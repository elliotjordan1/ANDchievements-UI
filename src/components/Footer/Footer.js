import React from 'react';
import Link from 'next/link';
import { FooterWrapper, FooterText, FooterDiv } from './styles';
import Logo from '../Logo';

const Header = () => {
  return (
    <FooterWrapper>
        <FooterDiv>
            <a href="https://and.digital/about/">
               <FooterText>About</FooterText>
            </a>
        </FooterDiv>
        <FooterDiv>
            <div>
               <FooterText>/</FooterText>
            </div>
        </FooterDiv>
        <FooterDiv>
            <a href="https://and.digital/contact/">
               <FooterText>Contact Us</FooterText>
            </a>
        </FooterDiv>
        <FooterDiv>
            <div>
               <FooterText>/</FooterText>
            </div>
        </FooterDiv>
        <FooterDiv>
            <a href="https://and.digital/join-us/">
               <FooterText>Join Us</FooterText>
            </a>
        </FooterDiv>
        <FooterDiv>
            <div>
               <FooterText>/</FooterText>
            </div>
        </FooterDiv>
        <FooterDiv>
            <a href="https://and.digital/">
               <FooterText>AND Digital</FooterText>
            </a>
        </FooterDiv>
    </FooterWrapper>
  );
};
export default Header;
