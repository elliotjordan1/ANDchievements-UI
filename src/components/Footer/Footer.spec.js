import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer component', () => {
  it('renders correctly', () => {
    const wrapper = render(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
