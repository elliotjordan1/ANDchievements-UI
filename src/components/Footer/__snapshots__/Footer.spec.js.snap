// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styles__FooterWrapper-sc-1x6z2gh-0 fhQJxy"
      >
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
            href="https://and.digital/about/"
          >
            About
          </a>
        </div>
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          >
            /
          </a>
        </div>
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
            href="https://and.digital/contact/"
          >
            Contact Us
          </a>
        </div>
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          >
            /
          </a>
        </div>
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
            href="https://and.digital/join-us/"
          >
            Join Us
          </a>
        </div>
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          >
            /
          </a>
        </div>
        <div
          class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
        >
          <a
            class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
            href="https://and.digital/"
          >
            AND Digital
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styles__FooterWrapper-sc-1x6z2gh-0 fhQJxy"
    >
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          href="https://and.digital/about/"
        >
          About
        </a>
      </div>
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
        >
          /
        </a>
      </div>
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          href="https://and.digital/contact/"
        >
          Contact Us
        </a>
      </div>
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
        >
          /
        </a>
      </div>
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          href="https://and.digital/join-us/"
        >
          Join Us
        </a>
      </div>
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
        >
          /
        </a>
      </div>
      <div
        class="styles__FooterDiv-sc-1x6z2gh-2 hmypNb"
      >
        <a
          class="styles__FooterText-sc-1x6z2gh-1 eRPpOX"
          href="https://and.digital/"
        >
          AND Digital
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
